### PR TITLE
[tune] Add util function to broadcast objects

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

This adds a util for closing over objects in ray tune. This cannot be done by ray.put since objectIDs are not serializable.

## Related issue number

https://github.com/ray-project/ray/issues/1453

### CODE REVIEW

File: doc/source/tune-config.rst [hunk 1]
```diff
@@ -69,6 +69,30 @@ By default, each random variable and grid search point is sampled once. To take
 
 For more information on variant generation, see `variant_generator.py <https://github.com/ray-project/ray/blob/master/python/ray/tune/variant_generator.py>`__.
 
+Broadcasting Objects
```
Comment (richardliaw) [inline doc/source/tune-config.rst]: `Handling Large Datasets/Broadcasting Objects`? (more relevance to supervised learning users)

**Review by richardliaw — COMMENTED** [review]
So this is just a workaround until named objects?

Generally, looks good. Would strongly prefer renaming of section in docs.

File: doc/source/tune-config.rst (hunk 1 above)
Comment (ericl) [reply doc/source/tune-config.rst]: Updated

**Comment by ericl** [comment]
Yeah, I think the right approach here is to have a tune util that we can deprecate later when ray gets proper support for the feature.

**Review by richardliaw — APPROVED** [review]

**Comment by ericl** [comment]
Tests look unrelated.

**Comment by ericl** [comment]
Tests look unrelated.

**Pull request merged**